ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -217,18 +217,17 @@ pipeline {
     }
     post {
         always { script {
-            // collect artifacts only if pr-test stage was executed.
-            // FIXME: this will break if we add an stage after skuba-test
-            if (pr_context == 'jenkins/skuba-test'){
-                archiveArtifacts(artifacts: "ci/infra/${PLATFORM}/terraform.tfstate", allowEmptyArchive: true)
-                archiveArtifacts(artifacts: "ci/infra/${PLATFORM}/terraform.tfvars.json", allowEmptyArchive: true)
-                archiveArtifacts(artifacts: 'testrunner.log', allowEmptyArchive: true)
-                archiveArtifacts(artifacts: 'ci/infra/testrunner/*.xml', allowEmptyArchive: true)
+            archiveArtifacts(artifacts: "ci/infra/${PLATFORM}/terraform.tfstate", allowEmptyArchive: true)
+            archiveArtifacts(artifacts: "ci/infra/${PLATFORM}/terraform.tfvars.json", allowEmptyArchive: true)
+            archiveArtifacts(artifacts: 'tfout.json', allowEmptyArchive: true)
```

Comment:
should this be moved inside the if?

Reply:
Yes, it could, but no needed, because of the allowEmpty.